pages/blog/creating-consistency-announcing-asyncapi-spectral-together.md

```diff
@@ -0,0 +1,206 @@
+---
+title: "Creating Consistency: Announcing AsyncAPI & Spectral – Together!"
+date: 2022-04-13T6:20:59+02:00
+type: Community
+tags: 
+  - API Governance 
+cover: /img/posts/creating-consistency-announcing-asyncapi-spectral-together/cover.webp
+authors:
+  - name: Stoplight
+    photo: /img/avatars/stoplight.webp
+    link: https://stoplight.io/
+    byline: We build API tooling
+excerpt: | 
+  The collaboration between Stoplight’s open-source linting tool, Spectral, and AsyncAPI is going to be a game changer for creating more consistency in your API program. Check out what comes with the release of the AsyncAPI ruleset!
+---
+
+# Creating Consistency: Announcing AsyncAPI & Spectral – Together!
+
+The collaboration between Stoplight’s open-source linting tool, Spectral, and AsyncAPI is going to be a game changer for creating more consistency in your API program. Check out what comes with the release of the AsyncAPI ruleset!
+
+Consistency is a top concern for companies across the globe.
+
+At Stoplight, we’re seeking to help your organization achieve consistency. We believe that API descriptions should be
+treated the same way as code. APIs should have style guides with rules, and be reviewed to ensure they are descriptive,
+readable, and concise for the developers who use them. Developer experience is, after all, another top concern for
+companies.
+
+So, we created our open-source JSON linting tool Spectral. It promotes standards with the flexibility for custom rule
+creation to validate and lint any JSON.
+
+For example, while an AsyncAPI or OpenAPI document might be entirely valid, it could be missing important fields like
+descriptions for parameters or have other problematic design issues. Spectral can warn about possible improvements to
+ensure API definitions can reach their full potential, without needing to have special code owners review every change
+to the documents.
+
+Although Spectral is mostly referenced in the context of OpenAPI, AsyncAPI (funded by Postman), has been an integral
```

Comment:
```suggestion
Although Spectral is mostly referenced in the context of OpenAPI, AsyncAPI has been an integral
```

I don't think we need to say it that way because it could sound as if we only have 1 org/company that has donated to the OSS AsyncAPI Initiative. 🤔   What do you think @derberg?

Reply:
Fair enough, removed :)